benchbuild/engine.py

```diff
@@ -0,0 +1,48 @@
+"""
+Orchestrate experiment execution.
+"""
+import abc
+from typing import Callable, List, Tuple, Type
+
+import attr
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'attr'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)